src/cbmc/bmc.h

```diff
@@ -149,7 +146,12 @@ class bmct:public safety_checkert
   friend class fault_localizationt;
 
 private:
-  symex_bmct &symex;
+  // We cannot use a reference member here
+  // because the initialization sequence would not be clean (Clang complains)
```

Comment:
What exactly does Clang complain about? I fail to see the problem with the original code...